packages/payload/src/utilities/validatePDF.ts

```diff
@@ -0,0 +1,17 @@
+export function validatePDF(buffer: Buffer) {
```

Comment:
We should always try to include a little jsdoc description when adding functions.